package.json

```diff
@@ -18,7 +18,6 @@
         "vitest": "vitest --run --coverage"
     },
     "dependencies": {
-        "@primevue/forms": "4.2.5",
```

Comment:
Shouldn't this dep stay? We're using formfields for Reference Select widget and could have a project that doesn't declare this dependency

Reply:
my bad, didn't realize that widget was here